db/schema.rb

```diff
@@ -17,6 +17,18 @@
   enable_extension "plpgsql"
   enable_extension "uuid-ossp"
 
+  create_table "disability_claims", force: :cascade do |t|
+    t.integer  "evss_id",                            null: false
+    t.json     "data",                               null: false
+    t.datetime "created_at",                         null: false
+    t.datetime "updated_at",                         null: false
+    t.string   "user_uuid",                          null: false
+    t.json     "list_data",          default: {},    null: false
+    t.boolean  "requested_decision", default: false, null: false
+  end
+
+  add_index "disability_claims", ["user_uuid"], name: "index_disability_claims_on_user_uuid", using: :btree
+
```

Comment:
@aub I must have run db:migrate instead of db:setup with the old schema. If ok, could you ok the change?